examples/webgl_tonemapping.html

```diff
@@ -162,6 +171,35 @@
 
 					} );
 
+				backgroundFolder.add( bgParams, 'rotation', - Math.PI, Math.PI, 0.1 )
+
+					.onChange( function ( value ) {
+
+						scene.backgroundRotation.y = value;
+						render();
+
+					} );
+
+				const environmentFolder = gui.addFolder( 'environment' );
+
+				environmentFolder.add( envParams, 'rotation', - Math.PI, Math.PI, 0.1 )
+
+					.onChange( function ( value ) {
+			
+						scene.environmentRotation.y = value;
+						render();
+			
+					} );
+
+				environmentFolder.add( envParams, 'intensity', 0, 2, 0.1 )
+
+					.onChange( function ( value ) {
+
+						scene.environmentIntensity = value;
+						render();
+
+					} );
+
```

Comment:
Thanks for creating the PR but I would try to keep `webgl_tonemapping` as focused as possible. The rotation feature is already demonstrated in `webgl_materials_envmaps` and that should be sufficient. 

The addition of `environmentIntensity` is okay, imo.

Reply:
I agree with what @Mugen87 said.

Reply:
Oh , as long as it's available to test in one example it's good, will close this for now